File: DesignPatternsExercises/java/FacadeDesignPattern/Exercise/StudentAccountVerification.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class StudentAccountVerification {
	private List<Integer> studentIdList = new ArrayList<>(Arrays.asList(2000, 2001, 2002, 2003, 2005, 2010));

	public boolean isStudentIdValid(int studentId) {
		if(studentIdList.contains(studentId)){
			return true;
		} else {
			System.out.println("Student ID is invalid");
			return false; 
		}
	}
}

File: DesignPatternsExercises/ruby/facade_design_pattern/solution/student_account_verification.rb
class StudentAccountVerification
  def initialize
    @student_id_list = [2000, 2001, 2002, 2003, 2005, 2010]
  end

  # def __________(7)__________(student_id)
  def student_id_valid?(student_id)
    return true if @student_id_list.include? student_id

    puts "Student ID #{student_id} is invalid"
    false
  end
end
